Snakefile

```diff
@@ -0,0 +1,37 @@
+configfile: "1_fetch/fetch_config.yaml"
+
+
+rule fetch_all:
+    input:
+        "1_fetch/in/pull_date.txt",
+        expand("{file}", file=config["sb_fetch.py"]["metadata_files"]),
+        expand("{file}", file=config["sb_fetch.py"]["obs_files"]),
+        expand("{file}", file=config["sb_fetch.py"]["driver_files"])
+
+# Lake information for 881 MNTOHA lakes
+rule fetch_metadata:
```

Comment:
with restructuring the config file, you could use just one rule (see https://github.com/jsadler2/lake-temperature-lstm-static/blob/ef32613bf46a67df06a80195472ac4518718be28/Snakefile#L18)

Reply:
Done! This is nice.